source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/update/UpdateAcquisitionContainer.test.tsx

```diff
@@ -141,6 +141,27 @@ describe('UpdateAcquisition container', () => {
     expect(popup).toBeVisible();
   });
 
+  it(`triggers the modal for contractor self-removal`, async () => {
+    mockUpdateAcquisitionFile.mockRejectedValue(
+      createAxiosError(
+        409,
+        'Contractors cannot remove themself from a file. Please contact the admin at pims@gov.bc.ca',
+        {
+          errorCode: UserOverrideCode.CONTRACTOR_SELFREMOVED,
+        },
+      ),
+    );
+    const { formikRef, findByText } = setup();
+
+    expect(formikRef.current).not.toBeNull();
+    await act(async () => formikRef.current?.submitForm());
+
+    const popup = await findByText(
+      /Contractors cannot remove themself from a file. Please contact the admin/i,
```

Comment:
nit: "Contractors cannot remove **themselves** from a file...."

I checked JIRA and the typo was there too - perhaps double check with @devinleighsmith ?